app.py

```diff
@@ -0,0 +1,12 @@
+from flask import Flask
+
+app = Flask(__name__)
+
+
+@app.route('/')
+def hello_world():  # put application's code here
+    return 'Hello World!'
+
+
```

Comment:
nit: Unnecessary line

Reply:
In python, double line spaces are preferable.